### PR TITLE
Adding a couple of scenarios for managed rules filtering

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

Adding 2 scenarios for the WebConsole OCP not showing managed rules

## Type of change

- New test feature file
- New test scenario added into existing feature file

## Testing steps

Not implemented steps are used, so no testing was done

## Checklist
* [x] Pylint passes for Python sources
* [x] sources has been pre-processed by Black
* [x] updated documentation wherever necessary

### CODE REVIEW

File: features/OCP_WebConsole/filter_managed_rules.feature [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Feature: Insights displayed on OCP WebConsole
+
+
+  Scenario: Managed rules are not shown in OCP WebConsole
+    Given console.redhat.com is accessible
+      And user U1 is already logged in into console.redhat.com
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) with at least one cluster C1
+      And the cluster C1 sent insights-operator archive to the external data pip
```
Comment (juandspy) [inline features/OCP_WebConsole/filter_managed_rules.feature:9]: should `pip`  be `pipeline`?

Comment (joselsegura) [reply features/OCP_WebConsole/filter_managed_rules.feature:9]: YES! I forgot than one

File: features/OCP_WebConsole/filter_managed_rules.feature [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      And exactly 1 non-managed issue have been detected for cluster C1
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And button named "Open Console" should be displayed
+     When user U1 click on "Open Console" button
+     Then new page named "Overview" with OCP WebConsole content should be displayed
+      And clickable "Insights" label should be displayed on that page
+      And information about 1 issues found should be displayed below "Insights" label
```
Comment (tisnik) [inline features/OCP_WebConsole/filter_managed_rules.feature:40]: issue

**Review by tisnik — APPROVED** [review]
just one nitpick

**Comment by tdosek** [comment]
Looks good to me.

File: features/OCP_WebConsole/filter_managed_rules.feature (hunk 2 above)
Comment (joselsegura) [reply features/OCP_WebConsole/filter_managed_rules.feature:40]: TY!

**Pull request merged**